apps/examples/security_test/security_api/security_api_ss_test.c

```diff
@@ -76,15 +75,19 @@ test_securestorage(void)
 	PrintBuffer(TEST_SS_PATH, output.data, output.length);
 
 	printf("  . SEC Get Secure Storage List ...\n");
-
-	if (0 != ss_get_list_secure_storage(hnd, &count, &list)) {
+	if (0 != ss_get_list_secure_storage(hnd, &list)) {
 		printf("Fail\n	! ss_get_list_secure_storage\n");
 		goto exit;
 	}
 	printf("ok\n");
 	printf("[%20s] [%8s]\n", "FILE NAME", "FILE ATTR");
-	for (i = 0; i < count; i++) {
-		printf("[%20s] [%08x]\n", list[i].name, list[i].attr);
+	// It will be paased why list will be null always.
+	{
```

Comment:
Is this open brace is needed ?

Reply:
I set checkpoint to implements and I want to share this issue to my partners.
Because the ss_get_list will have to implement.

Reply:
Ok.

Reply:
So, the brace will be removed when the function and utc will be updated.

Reply:
Ok, Got it.